smoketest.sh

```diff
@@ -47,6 +47,22 @@ function runDemoApps() {
         --pod cryostat-pod \
         --rm -d quay.io/andrewazores/quarkus-test:0.0.2
 
+    if [ -z "$CRYOSTAT_WEB_PORT" ]; then
+        local webPort="$(xpath -q -e 'project/properties/cryostat.itest.webPort/text()' pom.xml)"
+    else
+        local webPort="8181"
+    fi
+    if [ -z "$CRYOSTAT_DISABLE_SSL" ]; then
+        local protocol="https"
+    else
+        local protocol="http"
+    fi
+    podman run \
+        --name quarkus-test-plugin \
+        --pod cryostat-pod \
+        --env org.acme.CryostatService/mp-rest/url="${protocol}://localhost:${webPort}" \
+        --rm -d quarkus-test:0.0.3
```

Comment:
The image link is incomplete, it should be `quay.io/andrewazores/quarkus-test:0.0.3`

Reply:
Ah yes, thanks.